src/test/java/edu/cornell/kfs/sys/document/web/struts/CuFinancialMaintenanceDocumentActionTest.java

```diff
@@ -0,0 +1,199 @@
+        Maintainable maintainable = EasyMock.createMock(FinancialSystemMaintainable.class);
+        EasyMock.expect(maintainable.getDataObjectClass()).andStubReturn(dataObjectClass);
+        EasyMock.replay(maintainable);
+        return maintainable;
+    }
+
+    protected DocumentHeader createMockDocumentHeader(DocumentStatus documentStatus) {
+        DocumentHeader documentHeader = EasyMock.createMock(FinancialSystemDocumentHeader.class);
+        EasyMock.expect(documentHeader.getWorkflowDocument()).andStubReturn(
+                createMockWorkflowDocument(documentStatus));
+        EasyMock.replay(documentHeader);
+        return documentHeader;
+    }
+
+    protected WorkflowDocument createMockWorkflowDocument(DocumentStatus documentStatus) {
+        WorkflowDocument workflowDocument = EasyMock.createMock(WorkflowDocumentImpl.class);
+        EasyMock.expect(workflowDocument.getStatus()).andStubReturn(documentStatus);
+        EasyMock.expect(workflowDocument.isInitiated()).andStubReturn(DocumentStatus.INITIATED.equals(documentStatus));
+        EasyMock.expect(workflowDocument.isSaved()).andStubReturn(DocumentStatus.SAVED.equals(documentStatus));
+        EasyMock.expect(workflowDocument.isEnroute()).andStubReturn(DocumentStatus.ENROUTE.equals(documentStatus));
+        EasyMock.expect(workflowDocument.isException()).andStubReturn(DocumentStatus.EXCEPTION.equals(documentStatus));
+        EasyMock.expect(workflowDocument.isProcessed()).andStubReturn(DocumentStatus.PROCESSED.equals(documentStatus));
+        EasyMock.expect(workflowDocument.isFinal()).andStubReturn(DocumentStatus.FINAL.equals(documentStatus));
+        EasyMock.expect(workflowDocument.isCanceled()).andStubReturn(DocumentStatus.CANCELED.equals(documentStatus));
+        EasyMock.expect(workflowDocument.isDisapproved()).andStubReturn(DocumentStatus.DISAPPROVED.equals(documentStatus));
+        EasyMock.expect(workflowDocument.isRecalled()).andStubReturn(DocumentStatus.RECALLED.equals(documentStatus));
+        EasyMock.replay(workflowDocument);
+        return workflowDocument;
+    }
+
+    protected void setupMockNote(NoteType noteType) {
+        testNote = EasyMock.createMock(Note.class);
+        EasyMock.expect(testNote.getNoteType()).andStubReturn(
+                createMockNoteTypeBo(noteType));
+        EasyMock.expect(testNote.getNoteTypeCode()).andStubReturn(noteType.getCode());
+        EasyMock.replay(testNote);
+    }
+
+    protected org.kuali.rice.krad.bo.NoteType createMockNoteTypeBo(NoteType noteTypeEnum) {
+        org.kuali.rice.krad.bo.NoteType noteTypeBo = EasyMock.createMock(org.kuali.rice.krad.bo.NoteType.class);
+        EasyMock.expect(noteTypeBo.getNoteTypeCode()).andStubReturn(noteTypeEnum.getCode());
+        EasyMock.replay(noteTypeBo);
+        return noteTypeBo;
+    }
+
+
+
+    private static class TestCuFinancialMaintenanceDocumentAction extends CuFinancialMaintenanceDocumentAction {

```

Comment:
This override just expands the method visibility from protected to public. I suppose it's not strictly needed because the test is in the same package as the superclass (and hence can call the protected method); let me know if you think it's worth removing.